### PR TITLE
Add script to export all Studio logs

### PR DESCRIPTION
**Pull request by jesper7**
Changes:
- Add a Bash script to export all Studio logs and include it in the AMI by default

### CODE REVIEW

File: packer/studio-selfhosted.pkr.hcl [hunk 1]
```diff
@@ -120,6 +120,11 @@ build {
     destination = "/home/ubuntu/.studio_install/helm3.sh"
   }
 
+  provisioner "file" {
+    source      = "export-studio-logs.sh"
+    destination = "/usr/local/bin/export-studio-logs"
```
Comment (mjasion) [inline packer/studio-selfhosted.pkr.hcl:125]: I fixed the permission for GHA and there is 
```
==> amazon-ebs.source: Upload failed: scp: /usr/local/bin/export-studio-logs: Permission denied
```

https://github.com/iterative/studio-selfhosted/actions/runs/4606313140/jobs/8140343409?pr=48

**Review by mjasion — CHANGES_REQUESTED** [review]

File: packer/studio-selfhosted.pkr.hcl (hunk 1 above)
Comment (jesper7) [reply packer/studio-selfhosted.pkr.hcl]: Oh, I forgot that writing to `/usr/local/bin` requires root permission.

From the Packer file provisioned docs:
> Warning: You can only upload files to locations that the provisioning user (generally not root) has permission to access. Creating files in /tmp and using a shell provisioner to move them into the final location is the only way to upload files to root owned locations.

Pushed a commit which hopefully fixes it

File: packer/studio-selfhosted.pkr.hcl [hunk 2]
```diff
@@ -120,6 +119,15 @@ build {
     destination = "/home/ubuntu/.studio_install/helm3.sh"
   }
 
+  provisioner "file" {
+    source      = "export-studio-logs.sh"
+    destination = "/tmp/export-studio-logs"
+  }
+
+  provisioner "shell" {
+      inline = ["mv /tmp/export-studio-logs /usr/local/bin/export-studio-logs"]
```
Comment (mjasion) [inline packer/studio-selfhosted.pkr.hcl:128]: Add this to `setup_root.sh` :-)

**Review by mjasion — CHANGES_REQUESTED** [review]

File: packer/studio-selfhosted.pkr.hcl (hunk 2 above)
Comment (jesper7) [reply packer/studio-selfhosted.pkr.hcl:128]: Right, I'd forgot! Will do

**Review by mjasion — APPROVED** [review]

**Pull request merged**